src/docs/MLOS/references/Overview.adoc

```diff
@@ -1,9 +1,13 @@
-*Machine Learning Open Studio (ML-OS)* is an interactive graphical interface that enables developers and data scientists to quickly and easily build, train, and deploy machine learning models at any scale. It provides a rich set of generic machine learning tasks that can be connected together to build either basic or complex machine learning workflows for various use cases such as: fraud detection, text analysis, online offer recommendations, prediction of equipment failures, facial expression analysis, etc.
+*Proactive Machine Learning (PML)* is a complete DSML platform (Data Science and Machine Learning) including a ML Studio, AutoML, Data Science Orchestration and MLOps for the deployment,
+training, execution and scalability of artificial intelligence and machine learning models on
+any type of infrastructure. Created for data scientists and ML engineers, the solution is simple to use and accelerates machine learning algorithms toward production.
 
-image::ML_OS_overview.png[align=center]
-Machine Learning Open Studio is open source and allows easy task parallelization, running them on resources matching constraints (Multi-CPU, GPU, data locality, library).
+
+image::PML_overview.png[align=center]
+Proactive Machine Learning platform provides a rich set of generic machine learning tasks that can be connected together to build either basic or complex machine learning workflows for various use cases such as: fraud detection, text analysis, online offer recommendations, prediction of equipment failures, facial expression analysis, etc.
+PML workflows enable users to manage machine learning pipelines through development lifecycle and allow easy task parallelization, running them on resources matching constraints (Multi-CPU, GPU, data locality, library).
 
 
 image::ML-Open-Studio-ActiveEon.png[align=center]
 
-The Machine Learning Open Studio  can be tried for free online https://try.activeeon.com/studio/#workflows/templates/machine-learning[here^].
+The Proactive Machine Learning platform is open source and can be tried for free online https://try.activeeon.com/studio/#workflows/templates/machine-learning[here^].
```

Comment:
The Proactive Machine Learning platform is open source and can be tried for free online -> 
The Proactive Machine Learning platform is an open source solution under AGPL license (to be checked). This solution can be tested online without installation on our try platform